### ISSUE TITLE
Publish and update to aws-sign2 0.6.1

### ISSUE THREAD

**Issue by thedevelopnik** [request/request#2483]
When using Rollup.js, the packing process breaks on the request dependency [aws-sign2](https://github.com/request/aws-sign), because of a duplicate export.

```
Error parsing /node_modules/aws-sign2/index.js: Duplicate export 'queryStringToSign' (225:32) in /node_modules/aws-sign2/index.js
```

This issue has been fixed in aws-sign2, and there is an open issue on that repository to publish the new package. Right now anyone using request, or a package that uses request as a dependency (in our case Webdriver.io, node-gyp and selenium-standalone) cannot use Rollup (and possibly other similar products?) in their build process.

Fix should be as simple as publishing aws-sign2 and updating the package in request. Thank you!

**Comment by pastelsky** [comment]
Faced this issue today. A good number of libraries that depend on request cannot be used since rollup will simply fail to compile the project. Updating `aws-sign2` would certainly be helpful.

**Comment by AndrewKvalheim** [comment]
Referencing #2635

**Comment by aschrijver** [comment]
I am having this issue now, but since March there have not been new releases. Is this project dead?

**Comment by abhiram308** [comment]
The issue was fixed in version 0.7.0 of aws-sign. [Here](https://github.com/request/aws-sign/issues/7)'s the request. If you are getting the error when you include "request" package then you might be using the older version of request. I came across this issue when I was using request package version 0.81.0. I then upgraded it to 2.88.0 and it worked. Good luck!
